### ISSUE TITLE
Replace accessProfile with isAccessibleForFree

### ISSUE THREAD

**Issue by dkudeki** [dkudeki/bibframe2ef#4]
We almost implemented a version of this for EF2, but walked it back during some last minute changes meant to move us away from some use cases we had dropped. However, for many future runs of the metadata we may want to return to using `isAccessibleForFree`. It turns out that `accessProfile` is a set of dated values that may have some internal meaning, but are not helpful to the average user. There are four possible values: `open`, `google`, `page` and `page+lowres`. With this change, `open` would map to `isAccessibleForFree` being set to `True`, while the other three values would map to `False`. This change would put the implementation in line with the published EF2 documentation.

**Comment by dkudeki** [comment]
Looking more into this, I have learned a few things:

1) The mapping from `accessProfile` values to `isAccessibleForFree` values was briefly discussed in June 2018, but `isAccessibleForFree` was ultimately dropped in February 2020 shortly before the final run of EF2.0 data. At that time it was a simple mapping from `974r` values, where `pd` mapped to `True`, and everything else mapped to `False`. This is pretty simplistic and not that exciting, but it turns out this is an accurate* mapping of values, while the `accessProfile` mapping did not align with 

2) The `accessProfile` values are no longer present in the MARC-in-JSON metadata files provided to us, so any fields derived from `974a` will not be generated in the future, unless we are able to argue that they should be added back into the metadata files we receive. I don't see a convincing argument as to why they should be. They're old, outdated, and probably won't make much sense to end users.

3) `isAccessibleForFree` is present in the published [Extracted Features 3.0 JSON-LD schema](https://schemas.hathitrust.org/EF_Schema_v_3.0), while `accessProfile` is not. I view this as the most canonical documentation that I need to fall in line with. This was published after I removed `isAccessibleForFree` and added `accessProfile,` so it is technically more up-to-date. And since the metadata for `accessProfile` is no longer available, it is an easy schema to fall back in line with.

As a result, I have moved back to including `isAccessibleForFree` based on the `974r` value, and removed `accessProfile` entirely.

*Copyright status changes over time, and we may not always be in line, and `pdus` is a bit of a grey area, but this is largely true in a way it isn't for `accessProfile` values.
